PKGBUILD-python-standalone

```diff
@@ -9,7 +9,7 @@ arch=('any' OR 'x86_64' 'aarch64')
 groups=('blackarch' MORE_BLACKARCH_GROUPS_HERE)
 url='URL pointing to download section of package'
 license=('WHATEVER' OR 'custom:unknown')
-depends=('python')
+depends=('python' 'python-virtualenv')
```

Comment:
`python -m venv` doesn't use `python-virtualenv` so this additionnion is uneeded